srcpkgs/gendesk/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'gendesk'
 pkgname=gendesk
-version=1.0.6
+version=1.0.8
 revision=1
 build_style=go
 go_import_path=github.com/xyproto/gendesk
```

Comment:
Could you fix the `import_path` to match https://github.com/xyproto/gendesk/blob/main/go.mod ?